index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portafolio</title>
+</head>
+<body>
+    <nav class="nav">
+        <header class="logo">My Logo</header>
```

Comment:
The logo should a link with an a tag, making it a link is the best convention and also good for user experience. Also since it is not part of the menu options, kindly take it of the nav tag.